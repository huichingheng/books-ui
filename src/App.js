import React, { Component } from "react";
import "./App.css";
import Book from "./components/Book";
import Author from "./components/Author";

class App extends Component {
  render() {
    return (
      <div>
        <Book />
        <Author />
      </div>
    );
  }
}

export default App;
