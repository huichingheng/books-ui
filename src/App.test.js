import React from "react";
import ReactDOM from "react-dom";
import App from "./App";
import ShallowRender from "react-test-renderer/shallow";

it.skip("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<App />, div);
  ReactDOM.unmountComponentAtNode(div);
});

test("renders a div with some text", () => {
  const renderer = new ShallowRender();
  renderer.render(<App />);
  const output = renderer.getRenderOutput();

  expect(output).toMatchSnapshot();
});
