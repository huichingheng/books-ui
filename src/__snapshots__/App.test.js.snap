// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a div with some text 1`] = `
<div>
  <Book />
  <Author />
</div>
`;
